tests/Unit/PointwiseFunctions/AnalyticSolutions/EinsteinSolutions/Test_KerrSchild.cpp

```diff
@@ -0,0 +1,299 @@
+  const auto& g1 =
+      get<gr::Tags::SpatialMetric<3, Frame::Inertial, double>>(vars1);
+  const auto& dt_g1 =
+      get<gr::Tags::DtSpatialMetric<3, Frame::Inertial, double>>(vars1);
+  const auto& d_g1 =
+      get<EinsteinSolutions::KerrSchild::deriv_spatial_metric<double>>(vars1);
+
+  const auto vars2 = solution.solution(x2, t);
+  const auto& lapse2 =
+      get<gr::Tags::Lapse<3, Frame::Inertial, DataVector>>(vars2);
+  const auto& dt_lapse2 =
+      get<gr::Tags::DtLapse<3, Frame::Inertial, DataVector>>(vars2);
+  const auto& d_lapse2 =
+      get<EinsteinSolutions::KerrSchild::deriv_lapse<DataVector>>(vars2);
+  const auto& shift2 =
+      get<gr::Tags::Shift<3, Frame::Inertial, DataVector>>(vars2);
+  const auto& d_shift2 =
+      get<EinsteinSolutions::KerrSchild::deriv_shift<DataVector>>(vars2);
+  const auto& dt_shift2 =
+      get<gr::Tags::DtShift<3, Frame::Inertial, DataVector>>(vars2);
+  const auto& g2 =
+      get<gr::Tags::SpatialMetric<3, Frame::Inertial, DataVector>>(vars2);
+  const auto& dt_g2 =
+      get<gr::Tags::DtSpatialMetric<3, Frame::Inertial, DataVector>>(vars2);
+  const auto& d_g2 =
+      get<EinsteinSolutions::KerrSchild::deriv_spatial_metric<DataVector>>(
+          vars2);
+
+  check_tensor_doubles_equals_tensor_datavectors(lapse2, lapse1);
+  check_tensor_doubles_equals_tensor_datavectors(shift2, shift1);
+  check_tensor_doubles_equals_tensor_datavectors(g2, g1);
+  check_tensor_doubles_equals_tensor_datavectors(dt_lapse2, dt_lapse1);
+  check_tensor_doubles_equals_tensor_datavectors(dt_shift2, dt_shift1);
+  check_tensor_doubles_equals_tensor_datavectors(dt_g2, dt_g1);
+  check_tensor_doubles_equals_tensor_datavectors(d_lapse2, d_lapse1);
+  check_tensor_doubles_equals_tensor_datavectors(d_shift2, d_shift1);
+  check_tensor_doubles_equals_tensor_datavectors(d_g2, d_g1);
+}
+
+void test_serialize() {
+  EinsteinSolutions::KerrSchild solution(3.0, {{0.2, 0.3, 0.2}},
+                                         {{0.0, 3.0, 4.0}});
+  test_serialization(solution);
+}
+
+void test_copy_and_move() {
+  EinsteinSolutions::KerrSchild solution(3.0, {{0.2, 0.3, 0.2}},
+                                         {{0.0, 3.0, 4.0}});
+  test_copy_semantics(solution);
+  auto solution_copy = solution;
+  // clang-tidy: std::move of trivially copyable type
+  test_move_semantics(std::move(solution), solution_copy);  // NOLINT
+}
+
+void test_construct_from_options() {
+  Options<tmpl::list<EinsteinSolutions::KerrSchild>> opts("");
+  opts.parse(
+      "KerrSchild:\n"
+      "  Mass: 0.5\n"
+      "  Spin: [0.1,0.2,0.3]\n"
+      "  Center: [1.0,3.0,2.0]");
+  CHECK(
+      opts.get<EinsteinSolutions::KerrSchild>() ==
+      EinsteinSolutions::KerrSchild(0.5, {{0.1, 0.2, 0.3}}, {{1.0, 3.0, 2.0}}));
+}
+
+}  // namespace
+
+SPECTRE_TEST_CASE(
+    "Unit.PointwiseFunctions.AnalyticSolutions.EinsteinSolution.KerrSchild",
+    "[PointwiseFunctions][Unit]") {
+  test_schwarzschild<DataVector>(DataVector{1.0, 2.0, 3.0});
```

Comment:
since the DataVector and double are just used for size, let's set them to 0. or `nan` to make it obvious they're not used for anything